scripts/download-project-assets/download-project-assets.js

```diff
@@ -85,7 +86,12 @@ program
     await fs.rm(directory, { recursive: true, force: true });
     await fs.mkdir(directory);
 
-    await batchPromises(10, assets, (asset) => downloadAsset(asset, directory, assetsBar));
```

Comment:
Since we don't use `batchPromises` anymore, we can remove it from package.json and delete import statements.